packages/accounts-password/password_server.js

```diff
@@ -461,9 +461,10 @@ const pluckAddresses = (emails = []) => emails.map(email => email.address);
 // Method called by a user to request a password reset email. This is
 // the start of the reset process.
 Meteor.methods({forgotPassword: options => {
-  check(options, {email: String});
+  check(options, {email: String})
+
+  const user = Accounts.findUserByEmail(options.email, { fields: { emails: 1 } });
 
```

Comment:
Shouldn't the code to determine the message follow here, or am I missing something here?

Reply:
The error code is handled here.
It first checks that the flag is present if it isn't then it proceeds with the error message described.
https://github.com/meteor/meteor/blob/f5d3715431a2dd938afbf2f12d822476bd30b3fa/packages/accounts-password/password_server.js#L109